pkg/indexmanagement/scripts.go

```diff
@@ -35,18 +35,29 @@ from __future__ import print_function
 import json,sys
 r=json.load(sys.stdin)
 alias="${POLICY_MAPPING}-write"
-indices = [index for index in r if r[index]['aliases'][alias]['is_write_index']]
-if len(indices) > 0:
-  print(indices[0])
+try:
+  indices = [index for index in r if r[index]['aliases'][alias]['is_write_index']]
+  if len(indices) > 0:
+    print(indices[0])
+except Exception as e:
```

Comment:
doesn't this make it so a single bad `r[index]` could make it so we don't print anything? i feel like this is a little heavy handed

Reply:
Note this change was recommended by 2 other members of the team as cleaner to read in lieu of a block of "if" statements.  This will help us figure out why we have a single "bad" index.  I would expect it to fail outright or work.  We cant seem to understand why it sometimes fails and sometime succeeds.  The outcome on line 47 should allow us to see what was being processed.

Reply:
but.... i did realize the following evaluation will fail because it will not propagate the failure